src/organisms/cards/PolicyCard/CarrierLogo.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Image } from 'cloudinary-react';
+import styles from './policy_card.module.scss';
+
+export const CarrierLogo = ({ carrierLogo }) =>
+  <div className={styles['carrier-logo']}>
+    <Image
```

Comment:
@cisacke are we going to be using cloudinary to host these images? seems like they are being served up by rails, at least that's how it is in development

Reply:
@jmcolella unless there's a reason not to, I think this should be ok? I implemented it this way because it's how we do it in life-web, and I didn't want to use `Icon` and have to add all the carriers as svgs

Reply:
-- take this back, it does make sense to use the current implementation, I've made a PR for this [here](https://github.com/policygenius/athenaeum/pull/302)